Automattic/wp-calypso
# Migrate user item styles

#### Changes proposed in this Pull Request

Migrate styles used for user item (user avatar and name).

#### Testing instructions

Go to http://calypso.localhost:3000/devdocs/design/user-item and verify that the user item is styled.

##### Unstyled
<img width="187" alt="screen shot 2018-10-02 at 10 44 25 pm" src="https://user-images.githubusercontent.com/10561050/46372164-f3b9ee00-c6bc-11e8-8999-41cac76372c6.png">

##### Styled
<img width="220" alt="screen shot 2018-10-02 at 10 44 04 pm" src="https://user-images.githubusercontent.com/10561050/46372172-f74d7500-c6bc-11e8-8aab-c5aa87b1b530.png">

#27515
